index.js

```diff
@@ -20,12 +22,38 @@ const client = new NodeClient(clientOptions)
 const hsdHandler = async (req, res) => {
     try {
         const result = await client.execute('getnameresource', [req.params.name])
-        res.json({ result })
+        console.log(`Received result: ${result}`)
+        let resolved
+        if (result.result && result.result.records) {
+            const recs = result.result.records
+            for (let i = 0; i < recs.length; i++) {
+                if (recs[i].address) {
+                    resolved = recs[i].address
+                    break
+                }
+            }
```

Comment:
nit: just for improved readability

```suggestion
            const records = result.result.records
            for (let i = 0; i < records.length; i++) {
                if (records[i].address) {
                    resolved = records[i].address
                    break
                }
            }
```